css/style.css

```diff
@@ -150,48 +177,62 @@ a {
   text-decoration: underline;
   font-weight: bold;
 }
-a:hover { color: #d94f34; text-decoration:none }
+
+a:hover { color: #d94f34; text-decoration: none }
+
 p.center { text-align: center }
+
 p.buttons .button{ margin-bottom: 1em }
+
 p.buttons .button:last-child{ margin-bottom: 0 }
+
 .button.inverted.grey{ color: #767676 !important }
 
 body.content #top-bar{
   max-width: 1000px;
   margin: 0 auto !important;
 }
+
 body.content #top-bar .logo {
   margin-left: 0;
   margin-right: 0;
 }
+
 body.content #main article {
   max-width: 1000px !important;
   padding: 0 0.5em 1em;
   margin: 0 auto;
   text-align: justify;
 }
+
 body.content.readable #main article{
   font-size: 1.2em
 }
+
 @media only screen and (min-width: 1000px) {
   body.content #top-bar > div {
     padding-left: 0.5em !important;
     padding-right: 0 !important;
   }
 }
+
 body.content .buttons { margin-bottom: -1em }
+

```

Comment:
is it really necessary to have these blank lines? can you configure the linter to avoid requiring this?

Reply:
@nemesisdesign Definitely, i just thought it makes the css more readable and clear.. And some files were using a line empty before next rule and some were not so to make each of them similar. Do lemme know if i should undo this

Reply:
I like to have compact code, if you feel lines are too close to one another you should simply increase the `line-height` property of your editor, I really prefer if you don't add all these blank lines because they just make the code longer for me to read